overlord/devicestate/devicemgr.go

```diff
@@ -139,13 +140,24 @@ func (m *DeviceManager) Stop() {
 	m.runner.Stop()
 }
 
+func useStaging() bool {
+	return os.Getenv("SNAPPY_USE_STAGING_STORE") == "1"
+}
+
+func deviceAPIBaseURL() string {
+	if useStaging() {
+		return "https://myapps.developer.staging.ubuntu.com/identity/api/v1/"
```

Comment:
It's okay to postpone until after RTM, if you feel like this would delay you further.